### ISSUE TITLE
Please Update requirements.txt

### ISSUE THREAD

**Issue by elishafer** [rail-berkeley/serl#44]
Could you please update the requirements.txt (edit: in serl_launcher)?
gym is listed, but serl uses gymansium. Are there any other updates that need to be done to the file?

**Comment by elishafer** [comment]
pnput is also missing

**Comment by jianlanluo** [comment]
@youliangtan @charlesxu0124 I don't remember we used gymansium anywhere?

**Comment by youliangtan** [comment]
Good point. We are using both gym and gymnasium interchangeably (it's the same API), this should be more consistent, maybe should just stick to gym. 🤔 Will open a PR to address this update

**Comment by jianlanluo** [comment]
Oh ok, let's probably just stick with gym then?

what about pynput? should we also update the requirements.txt?

**Comment by youliangtan** [comment]
Updated, and all gymnasium has been converted to gym interface.

However, i will still keep gynasium being used here in the [mujoco renderer](https://github.com/rail-berkeley/serl/blob/6c320f6b836dbcb19a208b3e5a95a3bede8b8393/franka_sim/franka_sim/envs/panda_pick_gym_env.py#L140-L147). THis is only avail in gymnasium. Technically it is possible to do it in env, but it is not urgently required. refer to the comments.

TLDR: gym is mainly used. gymnaisum is still required. so install both. requirements.txt and setup.py are all updated

**Comment by elishafer** [comment]
@jianlanluo @youliangtan Thanks for addressing this so quickly

**Comment by elishafer** [comment]
@youliangtan @jianlanluo Are there any reasons for staying with gym instead of moving to gymnasium? [Gym's github page](https://github.com/openai/gym) states that they're no longer being maintained (last release was in Oct 2022)
